server/src/services/auth_RoleValidator.ts

```diff
@@ -0,0 +1,6 @@
+import { Role } from "../entities/role";
+import { User } from "../entities/user";
+
+export function auth_RoleValidator(user: User, roles: Role[]): boolean {
+    return [user.role].some((r) => roles.includes(r));
```

Comment:
you can just simplify this since we only have 1 user role to roles.includes(user.role);